analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.CSharp11.cs

```diff
@@ -83,14 +83,14 @@ void InterpolatedRawStringLiterals(string param)
         """;      // Noncompliant@-3
 
     string empty = "";
-    string x = $"""{empty}"""; // Noncompliant FP (string is still empty, should be compliant)
+    string x = $"""{empty}"""; // Noncompliant  string interpolation values are intentionally not evaluated
     x = $"""{empty}Test""";
     Foo(x);
 
     string emptyMultiline = """
```

Comment:
this is not really empty.

empty string AST:
![image](https://user-images.githubusercontent.com/38876598/201030118-cab7ce2b-4a18-4c4b-8771-1cd746a6acef.png)

empty line multi-string:
![image](https://user-images.githubusercontent.com/38876598/201030231-e34cbeb5-0ac5-488c-b353-cc8a03b0109b.png)

Reply:
Please check: https://github.com/SonarSource/sonar-dotnet/issues/6301#issuecomment-1311870738 and https://github.com/SonarSource/sonar-dotnet/issues/6301#issuecomment-1311871759